daml-lf/spec/daml-lf-1.rst

```diff
@@ -1191,6 +1240,22 @@ Then we define *well-formed expressions*. ::
     ——————————————————————————————————————————————————————————————— ExpCase
       Γ  ⊢  'case' e 'of' alt₁ | … | altₙ : σ
 
+      Γ  ⊢  τ  :  ⋆      Γ  ⊢ₑ  τ
+      Γ  ⊢  σ  :  ⋆
+      Γ  ⊢  e  :  τ
+    ——————————————————————————————————————————————————————————————— ExpThrow
+      Γ  ⊢  'throw' @τ @σ e  :  σ
```

Comment:
I think `throw` should take an `AnyException` instead of taking a polymorphic generic type. This would make it easier to use in the standard library (don't need a specialized throw method for each exception type, only need `toAnyException` and `fromAnyException` methods). (Additionally this allows one to (re-)throw `AnyException`, which is functionality that may come in handy at some point.)

Reply:
That's a good point. I'll change accordingly.